openlibrary/plugins/openlibrary/js/autocomplete.js

```diff
@@ -83,7 +78,7 @@
             autoFill: true,
             mustMatch: true,
             parse: function(text) {
-                var rows = eval(text).slice(0, options.max-1);
+                var rows = JSON.parse(text).slice(0, options.max-1);
```

Comment:
much better

Reply:
I felt a surge of energy burst through my hand as I first hit the backspace button; as if some great, restless force that had for so long been lurking in dread was at last, banished.